Tests/TokamakStaticHTMLTests/__Snapshots__/HTMLTests/testOptional.1.txt

```diff
@@ -160,6 +160,7 @@ height: 100%;
 
 "><svg style="width: 100%;
 height: 100%;
-overflow: visible; fill: rgba(255.0, 58.650000000000006, 48.45, 1.0);"><rect height="100%" stroke-width="0.0" width="100%" x="0.0" y="0.0"></rect></svg>
+overflow: visible; fill: rgba(255.0, 58.650000000000006, 48.45, 1.0);"><rect height="100%" stroke-width="0.0" width="100%" x="0.0" y="0.0"></rect>
+<defs></defs></svg>
```

Comment:
Does this `<defs></defs>` have any impact on rendering? If not, maybe our code could check if it's empty and avoid emitting it? IDK if that's the correct balance to strike here. I realize these additional checks may have impact on performance, but I'm for the smallest markup generated if possible. WDYT?

Reply:
No it's just a container element. I'll add a check 👍